velox/duckdb/conversion/DuckParser.cpp

```diff
@@ -306,6 +306,47 @@ static bool areAllChildrenConstant(const OperatorExpression& operExpr) {
   return true;
 }
 
+// Parse an CAST expression.
+std::shared_ptr<const core::IExpr> parseCastExpr(
```

Comment:
Can you move this below to its previous line number and reduce the diff?

Reply:
Yes, I have updated the code.